app/src/main/res/values/strings.xml

```diff
@@ -171,21 +171,34 @@ be temporary, and you can try again later.</li>
     <string name="settings_cookies_dialog_content">Selecting ‘Clear Now’ will clear all cookies and site data stored by Firefox for Fire TV. This may sign you out of websites and remove offline web content.</string>
 
     <string name="turbo_mode">Turbo Mode</string>
+    <!-- Displayed as notification when Turbo mode is turned on -->
+    <string name="turbo_mode_enabled">Turbo Mode enabled</string>
+    <!-- Displayed as notification when Turbo mode is turned off -->
+    <string name="turbo_mode_disabled">Turbo Mode disabled</string>
     <string name="onboarding_turbo_mode_title">Turbo Mode helps you browse faster</string>
     <string name="onboarding_turbo_mode_body">Turbo Mode automatically blocks trackers and ads to help get you where you\'re going faster. If there\'s a speed bump – like a site that isn\'t behaving the way you expect – just turn Turbo Mode off.</string>
     <string name="onboarding_turbo_mode_button_on">Keep Turbo Mode Enabled</string>
     <string name="onboarding_turbo_mode_button_off">Turn Turbo Mode Off</string>
```

Comment:
The context of onboarding is different than "turbo_mode_enable", where I think we're using it as opposites – why don't we just add a new string for the a11y use case?